packages/editor/src/client/routes/articleEditor.tsx

```diff
@@ -505,6 +505,7 @@ export function ArticleEditor({id}: ArticleEditorProps) {
       </Drawer>
       <Modal show={isPublishDialogOpen} size={'sm'} onHide={() => setPublishDialogOpen(false)}>
         <PublishArticlePanel
+          url={articleData?.article?.draft?.url || articleData?.article?.latest?.url}
```

Comment:
i suggest you put the url into the `metadata`. Then you'll always have the correct version when displaying it here.

Reply:
done